### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by Lulalaby**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Nathan Rajlich
```
Comment (LinusU) [inline LICENSE]: This seems incomplete, Nathan has committed not only in 2017, and there have been other contributors as well...

Comment (Lulalaby) [reply LICENSE]: Ah ok

Comment (Lulalaby) [reply LICENSE]: Could someone then provide a valid LICENSE.
GitHub License check fails/errors without it.

Comment (TooTallNate) [reply LICENSE]: Just update it to `2012 Nathan Rajlich, Linus Unnebäck and contributors` and it LGTM

**Comment by Lulalaby** [comment]
@TooTallNate still no merge? :D

**Comment by Lulalaby** [comment]
Hey what's the status?

**Comment by Lulalaby** [comment]
Ping

**Comment by Lulalaby** [comment]
Sad

**Comment by LinusU** [comment]
Sorry for never getting to this 😞 

I've fixed it here: b34cdc2

**Comment by Lulalaby** [comment]
XD

**Comment by Lulalaby** [comment]
All fine <3

**Pull request closed without merging**